swarm/network/stream/snapshot_retrieval_test.go

```diff
@@ -321,11 +326,14 @@ func runRetrievalTest(chunkCount int, nodeCount int) error {
 						log.Debug(fmt.Sprintf("File with root hash %x successfully retrieved", hash))
 					}
 				}
-				allSuccess = localSuccess
+				if !localSuccess {
```

Comment:
there should be error check on line 320

Reply:
that is not an error, but a `isEncrypted` boolean which is returned. I think it can be safely skipped for this test.